rs/messaging/src/routing/stream_handler/tests.rs

```diff
@@ -2384,10 +2388,11 @@ fn induct_stream_slices_partial_success() {
                 // ...a request to a missing canister @46 (on this subnet according to the
                 // routing table); this is expected to trigger a reject response...
                 Request(*REMOTE_CANISTER, *OTHER_LOCAL_CANISTER),
-                // ..a request not from `REMOTE_SUBNET` @47...
+                // ..a request not from `REMOTE_SUBNET` @47; this is also expected to trigger
+                // a reject respose...
                 Request(*LOCAL_CANISTER, *LOCAL_CANISTER),
                 // ...a request from a missing canister @48 (not anywhere according to the routing
-                // table); this is expected to be accepted but dropped...
+                // table); this is expected to be a reject response...
```

Comment:
Nit: These should all be "reject signal" rather than "reject response".

Same in the test case's doc comment.